components/dashboard/src/admin/TeamsSearch.tsx

```diff
@@ -0,0 +1,138 @@
+        return <Redirect to="/"/>
+    }
+
+    if (currentTeam) {
+        return <TeamDetail team={currentTeam} members={teamMembers[currentTeam.id]} />;
+    }
+
+    const search = async () => {
+        setSearching(true);
+        try {
+            const result = await getGitpodService().server.adminGetTeams({
+                searchTerm,
+                limit: 100,
+                orderBy: 'creationTime',
+                offset: 0,
+                orderDir: "desc"
+            })
+            setSearchResult(result);
+        } finally {
+            setSearching(false);
+        }
+    }
+    return <>
+        <div className="pt-8 flex">
+            <div className="flex justify-between w-full">
+                <div className="flex">
+                    <div className="py-4">
+                        <svg className={searching ? 'animate-spin' : ''} width="16" height="16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            <path fillRule="evenodd" clipRule="evenodd" d="M6 2a4 4 0 100 8 4 4 0 000-8zM0 6a6 6 0 1110.89 3.477l4.817 4.816a1 1 0 01-1.414 1.414l-4.816-4.816A6 6 0 010 6z" fill="#A8A29E" />
+                        </svg>
+                    </div>
+                    <input type="search" placeholder="Search Teams" onKeyDown={(k) => k.key === 'Enter' && search()} onChange={(v) => { setSearchTerm(v.target.value) }} />
+                </div>
+                <button disabled={searching} onClick={search}>Search</button>
+            </div>
+        </div>
+        <div className="flex flex-col space-y-2">
+            <div className="px-6 py-3 flex justify-between text-sm text-gray-400 border-t border-b border-gray-200 dark:border-gray-800 mb-2">
+                <div className="w-7/12">Name</div>
+                <div className="w-5/12 flex items-center"><span>Created</span>
+                    <svg xmlns="http://www.w3.org/2000/svg" fill="none" className="h-4 w-4" viewBox="0 0 16 16"><path fill="#A8A29E" fill-rule="evenodd" d="M13.366 8.234a.8.8 0 010 1.132l-4.8 4.8a.8.8 0 01-1.132 0l-4.8-4.8a.8.8 0 111.132-1.132L7.2 11.67V2.4a.8.8 0 111.6 0v9.269l3.434-3.435a.8.8 0 011.132 0z" clip-rule="evenodd" /></svg>
+                </div>
+
+
+            </div>
+            {searchResult.rows.map(team => <TeamResultItem team={team} members={teamMembers[team.id]} />)}
+        </div>
+    </>
+
+function TeamResultItem(props: { team: Team, members: TeamMemberInfo[]}) {
+    const { team, members } = props;
+    return (
+        <Link key={'pr-' + team.name} to={'/admin/teams/' + team.id} data-analytics='{"button_type":"sidebar_menu"}'>
+            <div className="rounded-xl whitespace-nowrap flex py-6 px-6 w-full justify-between hover:bg-gray-100 dark:hover:bg-gray-800 focus:bg-gitpod-kumquat-light group">
+                <div className="flex flex-col w-7/12 truncate">
+                    <div className="font-medium text-gray-800 dark:text-gray-100 truncate">{team.name}
```

Comment:
**issue**: Although the members count creates some visual balance, I'm already noticing a small delay in rendering with 16 teams on a not-so-fast network so I'm inclined to suggest dropping it for now following the concerns in https://github.com/gitpod-io/gitpod/pull/8402#discussion_r813946261. Let's cut some corners! ✂️ 

https://user-images.githubusercontent.com/120486/155563962-ed700cf2-cc8e-4bdb-b620-c4ac38149770.mov

Reply:
Yass